packages/eas-cli/src/user/expoSsoLauncher.ts

```diff
@@ -0,0 +1,82 @@
+import openBrowserAsync from 'better-opn';
+import http from 'http';
+import { Socket } from 'node:net';
+import querystring from 'querystring';
+
+import Log from '../log';
+
+export default (options: {
+  expoWebsiteUrl: string;
+  serverPort: number;
+}): { executeAuthFlow: () => Promise<string> } => {
```

Comment:
prefer to just make this pop the browser rather than return a function that pops the browser (since all callsites just call that function)

```suggestion
}): Promise<string> => {
```

Reply:
Done